vim_menutrans_helper.py

```diff
@@ -0,0 +1,196 @@
+import re
```

Comment:
The first line of a Python script should be like this:

```
#!/usr/bin/env python3
```

Also, please leave the second line as empty.